### ISSUE TITLE
Version 1.4 no visible HayLoftPlaceable

### ISSUE THREAD

**Issue by ghost** [J0hnHawk/FS17_WebStats#70]
Hello.
Please help me.
Our config change to xml and update to Webstats version 1.4.
Even though I put the HayLoftPlaceable.xml file into my settings, it will not appear anyway.
In the previous version, HayLoftPlaceable appeared because it was set in include / savegame.php:

`	// Platzierbares Wurzelfruchtlager
	if ($className == 'HayLoftPlaceable') {
		$location = 'HayLoftPlaceable';
		foreach ( $item as $node ) {
			$fillType = strval ( $node ['fillType'] );
			$fillLevel = intval ( $node ['fillLevel'] );
			addCommodity ( $fillType, $fillLevel, $location );
			$mapconfig = array_merge ( $mapconfig, array (
					'HayLoftPlaceable' => array (
							'position' => '663 115 -300',
							'showInProduction' => false,
							'input' => array (),
							'output' => array () 
					) 
			) );
		}
	}`

HayLoftPlaceable
https://ls-portal.eu/multi-interim-storage-v2-0/

Config file:
[HayLoftPlaceable.zip](https://github.com/J0hnHawk/FS17_WebStats/files/1700209/HayLoftPlaceable.zip)

Thanks JanSe

**Comment by ghost** [comment]
HayLoftPlaceable in Server/savegame/vehicles

`<item className="HayLoftPlaceable" modName="FS17_MultiLager" filename="$moddir$FS17_MultiLager/MultiLager.xml" position="663.95770263672 115.60565185547 -300.34704589844" rotation="0 3.0167360305786 0" age="160" price="200000"><node fillType="dryGrass_windrow" fillLevel="0"/><node fillType="pigFood" fillLevel="813634.625"/><node fillType="straw" fillLevel="747953.22965813"/><node fillType="dryGrass" fillLevel="0"/><node fillType="silage" fillLevel="506426.0625"/><node fillType="forage" fillLevel="2238845.5383483"/><node fillType="forage_mixing" fillLevel="0"/><node fillType="grass_windrow" fillLevel="704378.125"/></item>`

**Comment by J0hnHawk** [comment]
Hm, first your config looks for me...
I changed mod recognizing and loading the mod configuration together with switching the map config to xml. 
So it took a moment, but i think you only need to change 
`<item name="HayLoftPlaceable" locationType="storage" />`
to
`<item name="MultiLager" locationType="storage" />`

In line 65 of savegame.php my script now uses the filename instead of className.
`$filename = cleanFileName ( $item ['filename'] );`

I hope this is it?

**Comment by ghost** [comment]
Does not show :(
I really do not know :(
When the old script in the savegame.php was displayed right away.

`<?xml version="1.0" encoding="utf-8"?>
<placeable>
	<MOD name="FS17_MultiLager" version="5" author="Morice / Giants" link="http://www.farming-simulator.com/"></MOD>
	<item name="MultiLager" locationType="storage" />
	<l10n>
		<text name="MultiLager">
			<en>MIX SILO</en>	
			<de>MIX SILO</de>			
			<cz>MIX SILO</cz>
		</text>
	</l10n>
</placeable>`

**Comment by J0hnHawk** [comment]
I'll download the mod and test ist by my own in the afternoon today.

**Comment by ghost** [comment]
Thank you a lot. :)

**Comment by J0hnHawk** [comment]
I used your mod config above (Feb. 7. 22:13) and it works.
I placed the storage in sosnovka in a new savegame and put wheat in...
![grafik](https://user-images.githubusercontent.com/13779553/36013981-b617fec0-0d67-11e8-808f-2a4bc634a3be.png)
![screenshot-2018-2-9 sosnovka webstats](https://user-images.githubusercontent.com/13779553/36014003-da3e14d8-0d67-11e8-9bcd-6bbbdd2d2883.png)
![grafik](https://user-images.githubusercontent.com/13779553/36014032-04fb8b24-0d68-11e8-89e9-0462cebe985a.png)
